integration-tests/management-interface/src/test/java/io/quarkus/it/management/ManagementInterfaceTestCase.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.it.management;
```

Comment:
IMHO this test should show that the application endpoint access on another port has an effect on what is returned in health or metrics, some basic assertion that the response contains some text token is enough